docs/source/installation.rst

```diff
@@ -95,7 +93,6 @@ Windows
 3. Install GPkit
 ++++++++++++++++
   - Run ``pip install gpkit`` at an Anaconda Command Prompt.
-  - Run ``pip install pint`` for units support (recommended)

```

Comment:
All the CI machines have pint installed. What exactly do you want me to 'update accordingly'?